### PR TITLE
made links shorter by adding aliases

### LINKED ISSUE #131: Use aliases for the links to make them shorter
Use aliases for the links to make them shorter for 3D printed parts, Vendor BOM, recommended tools and Gasket SVG's for all the pages.

### PR DESCRIPTION
**Pull request by Venkat4real**
for the issue #131

### CODE REVIEW

File: content/en/v1.1/fan-box.adoc [hunk 1]
```diff
@@ -167,22 +167,22 @@ These are components that need to be purchased.  Prices fluctuate, and so are no
 | Count | Description | URL | Number in Package 
 | 1 
 | Sanyo Denki 9BMB12F201
-| https://octopart.com/9bmb12f201-sanyo+denki-24320259
+| https://octopart.com/9bmb12f201-sanyo+denki-24320259[Sanyo Denki ]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc]: Please trim the trailing white space

File: content/en/v1.1/fan-box.adoc [hunk 2]
```diff
@@ -216,31 +216,31 @@ These tools are recommended. URLs are for tools purchased and used in the buildi
...
-| https://www.harborfreight.com/micro-flush-cutter-90708.html
+| https://www.harborfreight.com/micro-flush-cutter-90708.html[Flush cutter]
 
 | X-acto knife
-| https://www.amazon.com/Xacto-X3201-N0-Precision-Knife/dp/B00004Z2TQ
+| https://www.amazon.com/Xacto-X3201-N0-Precision-Knife/dp/B00004Z2TQ[X-acto knife]
 
 | 3D Printer (note the size of the print bed for the fan box)
-| https://www.prusa3d.com/category/original-prusa-mk4/
+| https://www.prusa3d.com/category/original-prusa-mk4/[3D Printer ]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc]: Trim white space

File: content/en/v1.1/fan-box.adoc [hunk 3]
```diff
@@ -216,31 +216,31 @@ These tools are recommended. URLs are for tools purchased and used in the buildi
 | Description | URL
 
 | 4-3/4 In. Bent Long Nose Pliers
-| https://www.harborfreight.com/4-34-in-bent-long-nose-pliers-63819.html
+| https://www.harborfreight.com/4-34-in-bent-long-nose-pliers-63819.html[Long Nose Pliers]
 
 | A hex screwdriver for the m4 screws
-| https://www.amazon.com/gp/product/B007ICWAJC
+| https://www.amazon.com/gp/product/B007ICWAJC[hex screwdriver ]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc]: Trim white space

File: content/en/v1.1/fan-box.adoc [hunk 4]
```diff
@@ -167,22 +167,22 @@ These are components that need to be purchased.  Prices fluctuate, and so are no
...
 | 1
 | Dryer Vent Installation Tape
-| http://www.nashuatape.com/
+| http://www.nashuatape.com/[DryerTape]
 | 1
 
 | 1
 | BNC Cable Jack Connector
-| https://octopart.com/112517-amphenol+rf-50123148
+| https://octopart.com/112517-amphenol+rf-50123148[ Jack Connector]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc]: Trim white space

File: content/en/v1.1/fan-box.adoc [hunk 5]
```diff
@@ -216,31 +216,31 @@ These tools are recommended. URLs are for tools purchased and used in the buildi
...
-| https://www.prusa3d.com/category/original-prusa-mk4/
+| https://www.prusa3d.com/category/original-prusa-mk4/[3D Printer ]
 
 | #0 Phillips head screwdriver
-| https://www.homedepot.com/p/Husky-8-in-1-Screwdriver-with-LED-Light-232360016/301959976
+| https://www.homedepot.com/p/Husky-8-in-1-Screwdriver-with-LED-Light-232360016/301959976[head screwdriver]
 
 | CPAP hose cleaner (for maintenance)
-| https://www.amazon.com/Cleaning-DreamStation-Diameter-Stainless-Cleaner/dp/B08HLQV2VK/
+| https://www.amazon.com/Cleaning-DreamStation-Diameter-Stainless-Cleaner/dp/B08HLQV2VK/[ CPAP hose cleaner]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc]: Trim white space

File: content/en/v1.1/fan-box.adoc [hunk 6]
```diff
@@ -216,31 +216,31 @@ These tools are recommended. URLs are for tools purchased and used in the buildi
...
 
 | Cable Crimper
-| https://www.amazon.com/IWISS-Crimping-Swaging-Aluminum-Sleeves/dp/B00JW4X4TO/ref=sr_1_20?dchild=1&keywords=cable+crimping+tool&qid=1630860032&sr=8-20
+| https://www.amazon.com/IWISS-Crimping-Swaging-Aluminum-Sleeves/dp/B00JW4X4TO/ref=sr_1_20?dchild=1&keywords=cable+crimping+tool&qid=1630860032&sr=8-20[Cable Crimper]
 
 | 9/16 Nut Driver for the BNC
-| https://www.milwaukeetool.com/Products-Repository/North-America/Hand-Tools/Screwdrivers/48-22-2427
+| https://www.milwaukeetool.com/Products-Repository/North-America/Hand-Tools/Screwdrivers/48-22-2427[9/16 Nut Driver ]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc]: Trim white space

**Review by Burhan-Q — APPROVED** [review]
Minor changes requested, but otherwise looks good to go

File: content/en/v1.1/fan-box.adoc (hunk 1 above)
Comment (Venkat4real) [reply content/en/v1.1/fan-box.adoc]: Edited

File: content/en/v1.1/fan-box.adoc (hunk 2 above)
Comment (Venkat4real) [reply content/en/v1.1/fan-box.adoc]: Edited

File: content/en/v1.1/fan-box.adoc (hunk 3 above)
Comment (Venkat4real) [reply content/en/v1.1/fan-box.adoc]: Edited

File: content/en/v1.1/fan-box.adoc (hunk 4 above)
Comment (Venkat4real) [reply content/en/v1.1/fan-box.adoc]: Edited

File: content/en/v1.1/fan-box.adoc (hunk 5 above)
Comment (Venkat4real) [reply content/en/v1.1/fan-box.adoc]: edited

File: content/en/v1.1/fan-box.adoc (hunk 6 above)
Comment (Venkat4real) [reply content/en/v1.1/fan-box.adoc]: edited

**Comment by Venkat4real** [comment]
Hi @Burhan-Q thanks for the review. I made the changes suggested.

File: new 
Comment (Burhan-Q) [inline new :1]: I don't think you want to add this to the repo. Also, now that it's here, you probably need to remove this key

File: new .pub
Comment (Burhan-Q) [inline new .pub:1]: This should also not be included

**Review by Burhan-Q — CHANGES_REQUESTED** [review]
Remove SSH files. Revoke SSH private key to prevent someone else from using it (guaranteed it's already compromised).

**Comment by Venkat4real** [comment]
Sorry, removed the files now.

**Review by Burhan-Q — APPROVED** [review]

**Review by capsulecorplab — CHANGES_REQUESTED** [review]
Should remove [.vscode/settings.json](https://github.com/tetrabiodistributed/papra/pull/134/files#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357) since we're not using Antora. Otherwise, LGTM

**Comment by Venkat4real** [comment]
@capsulecorplab Removed the settings.json file.

**Comment by capsulecorplab** [comment]
Thanks @Venkat4real ! Though it looks like you closed the PR without merging. Would you mind re-opening?

**Comment by Venkat4real** [comment]
Hi Sean @capsulecorplab . I reopened the PR

**Review by capsulecorplab — APPROVED** [review]

**Pull request merged**